packages/components/src/templates/quickfact.stories.ts

```diff
@@ -16,6 +16,19 @@ export default {
   title: 'Templates/Quickfact'
 };
 
+export const QuickfactWithSummary = {
+  name: 'Quickfact with Summary',
+  render: () =>
+    html` <sd-quickfact class="max-w-[410px]">
+      <sd-icon name="content/people" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div slot="summary" class="flex flex-col sm:gap-4">
+        <div class="text-base font-normal leading-normal sm:text-xl">
+          Over 4,000 employees in 9 international locations
+        </div>
+      </div>
+    </sd-quickfact>`
+};
+
 export const QuickfactWithDescription = {
```

Comment:
looking at the figma file mentioned in the issue this PR is about, one template is still missing:
![image](https://github.com/user-attachments/assets/04fe39ff-91b0-4914-a9a1-3cf1e9985105)

is this intended?

Reply:
we don't have this variant at all in storybook... :/

Reply:
Hey @karlbaumhauer, just use the dev tools to see the mobile versions :) 

![image](https://github.com/user-attachments/assets/6ab1af9a-3a31-44fa-be02-557ce5587e50)